src/header/mod.rs

```diff
@@ -118,6 +118,8 @@ pub struct Headers {
     data: HashMap<HeaderName, MuCell<Item>>
 }
 
+const MAX_HEADERS_COUNT: u8 = 100;
```

Comment:
We need to restrict the total size because actually it's possible to consume a lot of memory just by *one* header, e.g. Cookie.